exercises/concept/logs-logs-logs/.docs/introduction.md

```diff
@@ -38,31 +38,31 @@ Since `rune` is just an alias for `int32`, printing a rune's type will yield `in
 ```go
 myRune := '¿'
 fmt.Printf("myRune type: %T\n", myRune)
-// Output: myRune type: int32
+// => myRune type: int32
```

Comment:
```suggestion
// Output: myRune type: int32
```

As discussed here https://github.com/exercism/go/issues/2202 when an instruction is a print statement, we actually want to keep the `Output:` wording.

Can you change this back in the other print statements?

Reply:
Sorry, my bad. Reverted changes in https://github.com/exercism/go/pull/2236/commits/c773cd8c1bac7a02048db3e92e7abd3620ee32b6.